scripts/install.sh

```
@@ -0,0 +1,158 @@
+      aws s3api create-bucket --bucket $CDK_PARAM_S3_BUCKET_NAME --region "$REGION" --create-bucket-configuration LocationConstraint="$REGION"
+    fi
+    aws s3api put-bucket-versioning --bucket $CDK_PARAM_S3_BUCKET_NAME --versioning-configuration Status=Enabled
+    aws s3api put-public-access-block --bucket $CDK_PARAM_S3_BUCKET_NAME --public-access-block-configuration BlockPublicAcls=true,IgnorePublicAcls=true,BlockPublicPolicy=true,RestrictPublicBuckets=true
+    echo "Bucket $CDK_PARAM_S3_BUCKET_NAME created."
+fi
+
+# CodeBuild 内の update-tenant.sh / provision-tenant.sh は zip 内に `cdk/` と `scripts/` が
+# あることを想定するため、TenkaCloud のリポジトリ構造 (server/ / scripts/) を zip 用に
+# `server` → `cdk` リネームした staging に並べてから zip する。
+STAGING=$(mktemp -d)
+trap "rm -rf '$STAGING'" EXIT
```

Shellcheck (0.11.0)

[warning] Use single quotes, otherwise this expands now rather than when signalled.

(SC2064)